chains/evm/contracts/executor/Executor.sol

```diff
@@ -161,17 +164,28 @@ contract Executor is Ownable2StepMsgSender, IExecutor {
 
   /// @notice Validates whether or not the executor can process the message and returns the fee required to do so.
   /// @param destChainSelector The destination chain selector.
+  /// @param requestedBlockDepth The requested block depth for the message. `0` indicates waiting for finality.
+  /// @param dataLength The length of the message data in bytes.
+  /// @param numberOfTokens The number of tokens being transferred in the message.
   /// @param ccvs The CCVs that are requested on source.
-  /// @return fee The fee required to execute the message.
+  /// @return usdCentsFee The USD denominated fee for the executor.
+  /// @return execGasCost The gas required to execute the message on destination, excluding pool/CCV/receiver gas.
+  /// @return execBytes The byte overhead required to execute the message on destination, excluding pool/CCV bytes.
   function getFee(
     uint64 destChainSelector,
-    Client.EVM2AnyMessage calldata, // message
+    uint16 requestedBlockDepth,
```

Comment:
Maybe not for this PR, but we should align our terminology for this. I've seen `finalityThreshold`, `finalityConfig`, and now `requestedBlockDepth`.